libnavigation-base/src/main/java/com/mapbox/navigation/base/utils/DecodeUtils.kt

```diff
@@ -122,6 +122,15 @@ object DecodeUtils {
         return stepsGeometryDecodeCache.getOrDecode(legStep.geometry(), precision)
     }
 
+    @JvmStatic
+    fun clearCache() {
+        synchronized(stepsGeometryDecodeCache) {
+            cachedRoutes.clear()
+            stepsGeometryDecodeCache.clear()
```

Comment:
`stepsGeometryDecodeCache.maxSize` should be set to 1 for consistency with `cachedRoutes`.